rkt/image_gc.go

```diff
@@ -21,6 +21,7 @@ import (
 
 	"github.com/coreos/rkt/common"
 	"github.com/coreos/rkt/pkg/lock"
+	pkgPod "github.com/coreos/rkt/pkg/pod"
```

Comment:
@yifan-gu Gotcha, let's leave it for now, this is too bikesheddy, and we can factor this afterwards.